bedrock/firefox/templates/firefox/welcome/page19.html

```diff
@@ -228,8 +228,13 @@
   <div class="mzp-l-content mzp-t-content-md mzp-u-centered">
     <h1 class="c-main-title">{{ main_title }}</h1>
     <p class="c-main-tagline">{{ main_tagline }}</p>
-    <p class="c-main-cta"><a href="{{ url('firefox.new') }}" class="mzp-c-button mzp-t-product mzp-t-xl" rel="external" data-cta-text="Update now">{{ cta_text }}</a></p>
+
+    <p class="c-main-cta" id="update-firefox"><a href="{{ url('firefox.download.thanks') }}" class="mzp-c-button mzp-t-product mzp-t-xl" id="update-client" rel="external" data-cta-text="Update now">{{ cta_text }}</a></p>
```

Comment:
**Issue:** this should use a `download_firefox_thanks()` helper, since that contain logic to show different download options to people on unsupported systems, such as Windows 7/8.

Reply:
```suggestion
    <p class="c-main-cta" id="update-firefox">
      {{ download_firefox_thanks(alt_copy=cta_text, button_class='mzp-t-xl') }}
    </p>
```